dataplicity/tags.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import unicode_literals
+
+import subprocess
+import re
+import logging
+
+log = logging.getLogger("agent")
+
+
+TAG_SCRIPT = "/home/dataplicity/dataplicity.tags"
+
+
+def get_tag_list():
+    """Run the dataplicity.tags script, get output as a list of tags"""
+    try:
+        output = subprocess.check_output(TAG_SCRIPT)
+    except Exception as error:
+        log.error(error)
+        return []
```

Comment:
If the script returns an error, wouldn't this result in all tags for that device being deleted? I doubt that is desired...

Reply:
not desired.

Reply:
Although these tags are reliant entirely on this script.  If the script breaks or is deleted, then it should remove the tags.  Fix the script, the tags will re-appear on the device.

Reply:
If the script is deleted, the tags should be removed.  If the script exists but spews an error code, it should be ignored.